src/measure_theory/integral/circle_integral.lean

```diff
@@ -470,7 +470,7 @@ calc ∥cauchy_power_series f c R n∥
 ... ≤ (2 * π)⁻¹ * (∫ θ : ℝ in 0..2*π, ∥f (circle_map c R θ)∥) * |R|⁻¹ ^ n :
   begin
     rcases eq_or_ne R 0 with rfl|hR,
-    { cases n; simp [real.two_pi_pos] },
+    { cases n; simp [-mul_inv_rev, real.two_pi_pos] },
```

Comment:
This suggests there is a missing `simp` lemma somewhere for dealing with the result of `mul_inv_rev`, will you add that in the follow-up PR?

Reply:
No unfortunately I don't understand what 's going on, apart from the fact that `mul_inv_rev` used to not be a simp lemma and it now is to match `mul_in_rev\0`. I will investigate further... but either way this will wait for another PR because it involves other lemmas.

Reply:
Note that the problem stems from the fact that we use `π`, not `\tau` 😜